com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/IsNanNode.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class IsNanNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "IsNan",
+            "Out = isnan(In);",
+            new ParameterDescriptor("In", TYPE.Bool, Usage.In),
```

Comment:
The input type for IsNan should be Vector

Reply:
fixed